lib/order.rb

```diff
@@ -1,18 +1,68 @@
+require 'csv'
+require_relative 'customer'
+
 module Grocery
+
   class Order
     attr_reader :id, :products
 
     def initialize(id, products)
       @id = id
       @products = products
+      @product_list = @products.split(";")
+      get_products(@product_list)
```

Comment:
I'm unsure how to interpret this line; it returns a value that doesn't get assigned to anything